airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/constants/GlobalDataSizeConstants.java

```diff
@@ -0,0 +1,12 @@
+package io.airbyte.integrations.destination.jdbc.constants;
+
+import io.aesy.datasize.ByteUnit.IEC;
+import io.aesy.datasize.DataSize;
+
+public interface GlobalDataSizeConstants {
```

Comment:
The main purpose of this java file is Constan Holder. So the interface fields, by default, is constants. If we will replace the keyword with class, we need to add additional keywords: `public static final ` and make the constructor private. That's the only reason. Anyway, we can replace it with class, if this will make more sense for you as a reviewer.